objects/oSupply/Create_0.gml

```diff
@@ -3,4 +3,10 @@ maxResourceCount = 0;
 resourceCount = 0;
 image_speed = 0;
 
+with(oGameController){
+	other.spriteList = ds_map_find_value(sprites,other.object_index);
+}
```

Comment:
Could just do `var spriteList = oGameController.sprites[? object_index];` and get rid of the `with` call?